numpy/lib/tests/test_function_base.py

```diff
@@ -1031,7 +1034,8 @@ def test_0d(self):
     def test_subclass(self):
         class SubClass(np.ndarray):
             pass
-        a = self.a.view(SubClass)
+
+        a = self._create_arrays()[0].view(SubClass)
```

Comment:
Similarly, the indexing is a little hard to read

```suggestion
        a_orig, _ = self._create_arrays().view(SubClass)
        a = a_orig.view(SubClass)
```